values/strings.xml

```diff
@@ -383,13 +383,14 @@
     <string name="buildinginfo_happiness_comp">Lowest happiness: %2$.0f%% (%1$s)</string>
     <string name="draftselector_titlereligion">Religion</string>
     <string name="draft_church01_title">Church</string>
-    <string name="draft_church01_text">A church.</string>
+    <string name="draft_church01_text">Provides religion and faith for inhabitants of your city.</string>
     <string name="draft_mosque00_title">Mosque</string>
-    <string name="draft_mosque00_text">A mosque.</string>
+    <string name="draft_mosque00_text">Provides religion and faith for inhabitants of your city.</string>
     <string name="settings_carsimulation">Car simulation</string>
     <string name="settings_weather">Weather</string>
+    <string name="topic_valentines">Happy Valentine\'s Day!</string>
     <string name="topic_christmas">Happy Christmas!</string>
-    <string name="topic_newyear">Happy new year</string>
+    <string name="topic_newyear">Happy new year %d</string> <!--%d should be 2022, 2023, etc.-->
```

Comment:
Consider making a feature request for this instead of modifying the translation.